src/_guides/libraries/create-library-packages.md

```diff
@@ -13,8 +13,6 @@ with a focus on the most common kind of package,
 TODO: Add coverage of packages that contain tools.
 {% endcomment %}
 
-[packages]: /guides/packages
```

Comment:
We used to try to make all our links use this format, but for short links, inline _does_ seem more natural.